src/SFML/Window/Win32/WglContext.cpp

```diff
@@ -460,6 +470,14 @@ void WglContext::updateSettingsFromPixelFormat()
         return;
     }
 
+    // Detect if we are running using the generic GDI implementation and warn
+    if (actualFormat.dwFlags & PFD_GENERIC_FORMAT)
+    {
+        m_isGeneric = true;
+
+        err() << "Warning: Detected \"Microsoft Corporation GDI Generic\" OpenGL implementation" << std::endl;
```

Comment:
Was "The current OpenGL implementation is not hardware-accelerated" misleading, or why did you not add it here?

Reply:
At least according to the Microsoft documentation, there is a theoretical possibility that the GDI implementation can also be hardware accelerated. I'm pretty sure this never happened in reality, but the new wording corresponds more to their documentation. Besides, whether accelerated or not, if you are using the GDI implementation, you are going to have a bad time anyway (OpenGL 1.1, no extensions, etc.) so this one warning line should be enough as an indicator.

Reply:
I added in a check for acceleration anyway. 😁